### PR TITLE
ML.NET home page readme updated with getting started resources

### PR DESCRIPTION
**Pull request by CESARDELATORRE**
ML.NET home page readme updated with additional getting started resources such as, Sample apps, community sample apps, ML.NET videos playlist at YouTube, Documentation, etc. plus some refactoring of the page so the info is clearer.

This is important info to be seen upfront for new .NET users coming to the ML.NET repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,22 +8,54 @@ ML.NET allows .NET developers to develop their own models and infuse custom mach
 
 ML.NET was originally developed in Microsoft Research, and evolved into a significant framework over the last decade and is used across many product groups in Microsoft like Windows, Bing, PowerPoint, Excel and more.
```
Comment (ebarsoum) [inline README.md]: It used for both training and production, should we add that it is a production quality ML framework?

File: README.md [hunk 2]
```diff
@@ -8,22 +8,54 @@ ML.NET allows .NET developers to develop their own models and infuse custom mach
...
 
-ML.NET enables machine learning tasks like classification (for example: support text classification, sentiment analysis) and regression (for example, price-prediction).
+ML.NET enables machine learning tasks like classification (for example: support text classification, sentiment analysis), regression (for example, price-prediction) and many other ML tasks such as anomaly detection, time-series-forecast, clustering, ranking, etc.
 
-Along with these ML capabilities, this first release of ML.NET also brings the first draft of .NET APIs for training models, using models for predictions, as well as the core components of this framework such as learning algorithms, transforms, and ML data structures.
+ML.NET also brings .NET APIs for training models, using models for predictions, as well as the core components of this framework such as learning algorithms, transforms, and ML data structures.
```
Comment (ebarsoum) [inline README.md]: Should we say that ML.NET is end-to-end ML platform, it bring a training pipeline that cover data reading, transform, learner...etc? We can add also a high level diagram as an image.

File: README.md [hunk 3]
```diff
@@ -101,10 +134,6 @@ Console.WriteLine("prediction: " + prediction.Prediction);
 ```
 A cookbook that shows how to use these APIs for a variety of existing and new scenarios can be found [here](docs/code/MlNetCookBook.md).
 
-## Samples
-
-We have a [repo of samples](https://github.com/dotnet/machinelearning-samples) that you can look at.
-
 ## License
 
 ML.NET is licensed under the [MIT license](LICENSE).
```
Comment (ebarsoum) [inline README.md]: should we explicitly mentioned that it is free to use commercially?

File: README.md (hunk 1 above)
Comment (CESARDELATORRE) [reply README.md]: Agree. I modified the text including that perspective. 👍

File: README.md (hunk 3 above)
Comment (CESARDELATORRE) [reply README.md]: Done! 👍

**Pull request merged**